src/components/common/charts/styles/chartStyles.css

```
@@ -728,7 +716,7 @@
 
 .chart-pie:hover {
   transform: translateY(-2px) scale(1.02);
-  filter: drop-shadow(0 3px 6px rgba(0, 0, 0, 0.5));
+  filter: drop_shadow(0 3px 6px rgba(0, 0, 0, 0.5));
```

Biome (1.9.4)

[error] Unexpected unknown function: drop_shadow

Use a known function instead.
See MDN web docs for more details.

(lint/correctness/noUnknownFunction)